### ISSUE TITLE
Addressing Public Comments and Future Features

### ISSUE THREAD

**Issue by bradrydzewski** [indieopensource/paid-license#5]
Would you consider a pull request that adds language to clarify the license is not contingent upon future functionality? There is some prior art in the [Atlassian Agreement (section 7.29)](https://www.atlassian.com/legal/software-license-agreement).

> ### <a id="Future_Functionality"></a>Future Functionality
> 
> _Customer_ agrees payment is not conditional on delivery of future functionality or features, or dependent on any oral or written public comments made by _Developer_ regarding future functionality or features.

I realize this may feel unnecessary, however, I have started identifying patterns of common misconceptions and the right to request features is one of them.

**Comment by kemitchell** [comment]
Indeed, the backstop is here, in the boilerplate: https://github.com/indieopensource/paid-license/blob/f1c902a04e5bffe148935e4f472d334faa8c570b/terms.md#whole-agreement

**Comment by kemitchell** [comment]
I've really got to upgrade the tooling for this one to build off the Markdown. This could've been a PR if I had. My bad. Swamped.

**Comment by bradrydzewski** [comment]
I would be happy to send a PR that upgrades the tooling.  Do you have a repository I can use for reference?

**Comment by kemitchell** [comment]
I did it. Master builds off `terms.md` now. I need to a versioned release.

**Comment by kemitchell** [comment]
@bradrydzewski insofar as this is really a _messaging_ concern, do you think it might be more appropriate to put the clarification on the order form?

**Comment by bradrydzewski** [comment]
I definitely agree this is a messaging concern.

I suspect Atlassian added this to their terms for the same reason I would like to add to my terms. Customers get upset when they think they are not getting what they paid for, and sometimes these individuals get downright rude. Setting these expectations in the terms makes our stance crystal clear and leaves little room for disagreement. I see this as a way to diffuse the situation because it would have been spelled out in plain english in the terms, to which they agreed.

Adding this to the order is not a bad idea, however, our terms are posted to our website which means they may be evaluated prior to the order being generated.

With that being said I understand why you might prefer to keep this messaging out of the base template. I could use this as the foundation for my terms and slightly tweak for my needs. I will email you to discuss this a bit further.

**Comment by kemitchell** [comment]
I suspect license terms are often evaluated by different people than order forms, and that the latter, not the former, produce the issues you and Atlassian have seen.

A legal noop with relationship value in legal terms gets used as a gotcha.  You end up pointing back to terms the person complaining never read and saying, "See! We covered this." But that's far less compelling than "here it is half a page above your signature on the two-pager setting out our deal".

**Comment by bradrydzewski** [comment]
It really depends. I have a lot of customers that pay online with credit card using our stripe checkout form plus clickwrap agreement. For smaller deals it is not uncommon for the engineer or engineering manager to charge directly to their corporate card. The terms are the only place I can think of where everyone will see this messaging, regardless of payment process.

It may be true that sending a link to my terms is ineffective, but without the ability to experiment there is no way for me to say for sure. I personally draw a lot of inspiration from Atlassian. Atlassian has gone to great lengths to create a touchless enterprise sales process [1] that serves companies of all sizes, and my hunch is the messaging in their terms is intentional with the goal of mitigating some of the issues I'm seeing.

It sounds like I might be asking for customization above and beyond your goals for this repository. Lets close this for now and move the discussion to email.

[1] https://www.intercom.com/blog/podcasts/scale-how-atlassian-built-a-20-billion-dollar-company-with-no-sales-team/

**Comment by kemitchell** [comment]
> It really depends. I have a lot of customers that pay online with credit card using our stripe checkout form plus clickwrap agreement.

No, no. Don't close. You've got a good point here.
